scripts/base_network.py

```diff
@@ -280,6 +291,25 @@ def _set_electrical_parameters_transformers(transformers):
     return transformers
 
 
+def _set_electrical_parameters_converters(converters):
+    p_max_pu = snakemake.config["links"].get("p_max_pu", 1.0)
+    converters["p_max_pu"] = p_max_pu
+    converters["p_min_pu"] = -p_max_pu
+
+    converters["p_nom"] = 2000  # [MW]?
+
+    # Converters are combined with links
+    converters["under_construction"] = False
+    converters["underground"] = False
+
+    return converters
+
+
+# TODO Do we need such a cross-border function?
+# def _replace_b2b_converter_at_country_border_by_link(n):
```

Comment:
Aren't converters already links?

Reply:
The function cited in the TODO is a single cross-border converted that by some reasons had to be transformed into a link in the PyPSA-Eur model. That looks like quite a specific case, and I think we can remove this TODO safely.

Reply:
Ok